README.md

```diff
@@ -55,24 +55,22 @@ $ fake_sqs --help
 ```
 
 By default, FakeSQS uses an in-memory database (just a hash actually). To make
-it persistant, run with:
+it persistent, run with:
 
 ```
 $ fake_sqs --database /path/to/database.yml
 ```
 
 Messages are not persisted, just the queues.
 
-This is an example of how to configure the official [aws-sdk gem] [aws-sdk], to
-let it talk to Fake SQS.
+This is an example of how to configure the [AWS SDK for Ruby] [aws-sdk], to let
+it talk to Fake SQS.
 
 ``` ruby
-AWS.config(
-  :use_ssl           => false,
-  :sqs_endpoint      => "localhost",
-  :sqs_port          => 4568,
```

Comment:
`sqs_port` was removed in v1 -> v2, requiring fully qualified HTTP/HTTPS URIs to be configured in the endpoint setting